packages/fxa-auth-server/lib/payments/iap/apple-app-store/types/api-client.ts

```diff
@@ -0,0 +1,187 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2021 August Heegaard
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+// Below types are copied from 'app-store-server-api/dist/types/Models.d.ts' (with 'declare'
+// keyword omitted) as only the enum types (a minority) are available in
+// 'app-store-server-api/dist/cjs/', and TS in ./node_modules are not compiled into JS
+// along with the fxa source code.
```

Comment:
Maybe a TODO here with a reference to the other library that this can be removed when they export it?

Reply:
The PR I opened in the app-store-client-api repo was merged and the maintainer tagged a new release, so I plan to update the package in this PR and remove this file (i.e. close #12579 at the same time).